introduction.md

```diff
@@ -3,5 +3,8 @@ I'm form cartagena, Colombia,
 I'm a Software developer and I'm here 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?